javers-persistence-mongo/src/main/java/org/javers/repository/mongo/MongoRepository.java

```diff
@@ -254,23 +283,35 @@ private Bson applyQueryParams(Bson query, QueryParams queryParams) {
     }
 
     private Bson addFromDateFiler(Bson query, LocalDateTime from) {
-        Bson filter = new BasicDBObject(COMMIT_DATE, new BasicDBObject("$gte", DateTypeCoreAdapters.serialize(from)));
-        return Filters.and(query, filter);
+        return Filters.and(query, createFromDateQuery(from));
+    }
+
+    private Bson createFromDateQuery(LocalDateTime from) {
+        return Filters.gte(COMMIT_DATE, DateTypeCoreAdapters.serialize(from));
     }
 
     private Bson addToDateFilter(Bson query, LocalDateTime to) {
-        Bson filter = new BasicDBObject(COMMIT_DATE, new BasicDBObject("$lte", DateTypeCoreAdapters.serialize(to)));
-        return Filters.and(query, filter);
+        return Filters.and(query, createToDateQuery(to));
+    }
+
+    private Bson createToDateQuery(LocalDateTime to) {
+        return Filters.lte(COMMIT_DATE, DateTypeCoreAdapters.serialize(to));
     }
 
     private Bson addCommitIdFilter(Bson query, CommitId commitId) {
-        Bson filter = new BasicDBObject(COMMIT_ID, commitId.valueAsNumber().doubleValue());
-        return Filters.and(query, filter);
+        return Filters.and(query, createCommitIdQuery(commitId));
```

Comment:
ja bym inlineował, bo to jedyne użycie `createCommitIdQuery`, ale to kwestia gustu, nie upieram się